runner/src/server/plugins/engine/pageControllers/RepeatingFieldPageController.ts

```diff
@@ -21,17 +21,17 @@ function isInputType(component) {
 
 export class RepeatingFieldPageController extends PageController {
   summary: RepeatingSummaryPageController;
-  inputComponent: FormComponent;
+  inputComponent: FormComponent[];
```

Comment:
array should be plural

Reply:
Ideally remove

Reply:
reverted this back so its no longer an array.